unittests/util/Fixtures.h

```diff
@@ -123,31 +123,32 @@ struct AsyncAsioContextTest : virtual public NoLoggerFixture
     AsyncAsioContextTest()
     {
         work.emplace(ctx);  // make sure ctx does not stop on its own
+        runner.emplace([&] { ctx.run(); });
```

Comment:
This could be a `boost::asio::thread_pool` of 1 thread perhaps.

Reply:
Clio interfaces often requires io_context. Currently , there is no place using this fixture.